components/hero/Container.tsx

```diff
@@ -0,0 +1,16 @@
+import clsx from "clsx"
+
+export function Container({
```

Comment:
I think there's an existing wrapper component which I think serves this same purpose. Is there a reason why you created this separately?